### ISSUE TITLE
Obfuscating Data in Log issue

### ISSUE THREAD

**Issue by robertofrontado** [Instabug/Instabug-iOS#164]
Hello,

I'm using the following method to obfuscate a request before logging it:

```swift
Instabug.setNetworkLogRequestObfuscationHandler { (request) -> URLRequest in
  return self.obfuscateRequestBody(request: request)
}
```

So I'm trying to obfuscate some parameters that are in the `httpBody` of a request, but for some reason `httpBody` is always `nil` at this point,

Any idea?

**Comment by Kmohamed** [comment]
@robertofrontado let me check it and get back to you.

**Comment by robertofrontado** [comment]
That was fast @Kmohamed, thanks 😄

**Comment by Kmohamed** [comment]
@robertofrontado 
Actually it's a bug and it will fixed in the next release. If this bug will block you I can send you a custom build.

**Comment by robertofrontado** [comment]
Okk, I think I can wait until the next release, thank you!

**Comment by Kmohamed** [comment]
I will notify you once is got released.

**Comment by Kmohamed** [comment]
@robertofrontado we just released v7.2.9 which contains bug fix
